pkg/apis/abac/v0/register.go

```diff
@@ -40,9 +41,12 @@ func init() {
 }
 
 var (
+	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
 	// TODO: move SchemeBuilder with zz_generated.deepcopy.go to k8s.io/api.
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
```

Comment:
Can we move this line down to stay with localSchemeBuilder and AddToScheme?

Reply:
done